### DISCUSSION TITLE
how to add a 3D model to the config.json file located at model-viewer/packages/render-fidelity-tools/test.

### DISCUSSION THREAD

**Discussion by Jayy0906** [google/model-viewer#4328] [Q&A]
I want to add my own 3D model to the render fidelity tool and use different web renders such as [three.js], [filament.js], [babylon.js], [gltf-sample-viewer] like https://modelviewer.dev/fidelity/ did, to test it out and determine its dB values.

And how to configure render fidelity tools' config.json file's (model-viewer\packages\render-fidelity-tools\test\config.json) orbit, target, and dimensions.

**Comment by elalish** [comment]
As in, you want to open a PR to add this model to our repo? If it's available under an open license, that would be fine. Can we see a screenshot of the model?

If you run `npm run serve` at the project root, you can go to http://127.0.0.1:8080/packages/render-fidelity-tools/test/renderers/model-viewer/ to test our your scenario - just reload and select your case from the drop down. The configuration largely follows model-viewer's API, except there is no auto-centering.
